src_web/kamaho-shokusu/src/Policy/SystemReportPolicy.php

```
@@ -0,0 +1,47 @@
+
+use App\Domain\ValueObject\UserRole;
+use Authorization\IdentityInterface;
+
+/**
+ * システムレポート用認可ポリシー
+ *
+ * システム管理者（i_admin === 3）のみアクセスを許可する。
+ */
+class SystemReportPolicy
+{
+    public function canIndex(?IdentityInterface $user, \App\Controller\SystemReportController $resource): bool
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$resource'. (undefined)

(UnusedFormalParameter)